<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingProbeWidget</class>
 <widget class="QWidget" name="SettingProbeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>322</width>
    <height>201</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ProbeSettingWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_workspaceScale"/>
   </item>
   <item row="4" column="3">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Surface Sampling
Hit Threshold</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Enable
Haptics</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Haptics Mode</string>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>Haptic Damping
Force</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QCheckBox" name="checkBox_enableHaptics">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Workspace
Scale</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Silhouette Scale</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Paint Mode</string>
     </property>
    </widget>
   </item>
   <item row="5" column="4">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_hapticSpringForce">
     <property name="decimals">
      <number>2</number>
     </property>
     <property name="maximum">
      <double>4000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_silhouetteScale"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Probe Scale</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_paintScale"/>
   </item>
   <item row="0" column="1">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_probeScale"/>
   </item>
   <item row="3" column="4">
    <widget class="QComboBox" name="comboBox_hapticMode"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Paint Scale</string>
     </property>
    </widget>
   </item>
   <item row="6" column="4">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_hapticDampingForce">
     <property name="decimals">
      <number>2</number>
     </property>
     <property name="maximum">
      <double>4000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="5">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="comboBox_paintMode"/>
   </item>
   <item row="5" column="3">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Haptic Spring
Force</string>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_hapticSurfaceSamplingHitThreshold">
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.001000000000000</double>
     </property>
    </widget>
   </item>
   <item row="3" column="2" rowspan="4">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>Selected
Group</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QSpinBox" name="spinBox_selectedGroup">
     <property name="maximum">
      <number>9</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
